starknet_py/utils/typed_data.py

```diff
@@ -235,13 +295,28 @@ def escape(s: str, revision: Revision) -> str:
     return f'"{s}"'
 
 
+def prepare_selector(name: str) -> int:
+    try:
+        return int(_to_rpc_felt(name), 16)
+    except ValueError:
+        return get_selector_from_name(name)
+
+
+class BasicType(Enum):
+    FELT = "felt"
+    SELECTOR = "selector"
+    MERKLE_TREE = "merkletree"
+    SHORT_STRING = "shortstring"
+
+
 # pylint: disable=unused-argument
 # pylint: disable=no-self-use
 
 
 class ParameterSchema(Schema):
     name = fields.String(data_key="name", required=True)
     type = fields.String(data_key="type", required=True)
+    contains = fields.String(data_key="contains", required=False)
```

Comment:
Should it be required?

Reply:
Could you please clarify your question (because `contains` is actually not required)? `contains` appears in merkle tree parameter, in standard one it's not present - thats's why it's optional.

Reply:
Never mind, for some reason I saw `True` there, all good here.